### DISCUSSION TITLE
Sales().get_order_metrics() not accepting my interval date

### DISCUSSION THREAD

**Discussion by zesto01** [saleweaver/python-amazon-sp-api#678] [Q&A]
Hi, so I tried using Sales endpoint

`interval='2022-01-02T23:21:20Z+07:00-2022-08-02T23:21:20Z+07:00'`
`Sales(creds=creds).get_order_metrics(interval, Granularity.DAY, granularityTimeZone='US/Central')`

But I keep getting an error with the needed regex patterns 
I also tried what's in the example and it still didn't work.
`interval='2018-09-01T00:00:00-07:00–2018-09-04T00:00:00-07:00'`

My error:
`SellingApiBadRequestException: [{'code': 'InvalidInput', 'message': '{getOrderMetrics.arg1=must match "[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\\.[0-9]{3})?(([+-]\\d\\d:\\d\\d)|Z)(\\[[a-zA-Z/_]+\\])?--[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\\.[0-9]{3})?(([+-]\\d\\d:\\d\\d)|Z)(\\[[a-zA-Z/_]+\\])?"}', 'details': ''}]`

**Comment by atetz** [comment]
While stumbling upon the same error I found out I was setting the interval with the wrong type (str instead of tuple). 
The interval takes a tuple. For example:
interval = ("2022-10-05T00:00:00+02:00", "2022-10-12T00:00:00+02:00")

I also noticed the interval takes a tuple with datetime objects:
interval = (datetime.now(timezone.utc) - timedelta(days=7)), datetime.now(timezone.utc)

The solution is also mentioned here #407
